src/pages/AuthPopup/AuthPopupElements.jsx

```diff
@@ -41,8 +41,8 @@ export const PopupContainer = styled.div`
 
 export const CloseButton = styled(ImCross)`
     position: absolute;
-    top: 20px;
-    right: 20px;
+    top: 6px;
+    right: 6px;
```

Comment:
before:
![image](https://github.com/thecyberworld/TheCyberHUB/assets/76536506/980b36e0-5302-4429-849d-6d50c49318cc)

after:
![image](https://github.com/thecyberworld/TheCyberHUB/assets/76536506/ea03efe0-2fa0-4953-8123-c6fd8d96b4cc)

FeedPagePost.jsx , it looks there too close to the border, I like the previous option. what do you think?

Reply:
yes you are right it is surely too close , but the first one is too far , i think in the middle will be perfect

Reply:
yes you are right i think it is too close but the first one is too far , i think in the middle will be perfect